packages/twenty-server/src/engine/api/graphql/graphql-config/graphql-config.service.ts

```diff
@@ -141,8 +141,10 @@ export class GraphQLConfigService
     // Create a new contextId for each request
     const contextId = ContextIdFactory.create();
 
-    // Register the request in the contextId
-    this.moduleRef.registerRequestByContextId(context.req, contextId);
+    if (this.moduleRef.registerRequestByContextId) {
```

Comment:
In which case this is not defined?

Reply:
It's actually check in the official libraries, just did it too just in case
https://github.com/nestjs/bull/blob/4ed326178b889bd0f851cad736e03a1a0a1667c5/packages/bullmq/lib/bull.explorer.ts#L166